org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,158 @@
+{
+    "app-id" : "org.arrl.trustedqsl",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Locale" ],
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
+	"--filesystem=home"
+     ],
+    "modules" : [
+        {
+            "name" : "openssl",
```

Comment:
openssl is in the runtime.

Reply:
Openssl *development* is not in the runtime.

Reply:
no package need to build openssl 3.